CONTRIBUTING.md

```diff
@@ -239,134 +134,13 @@ Each plugin submitted to the plugins list should have the following:
-   structure the content.
-3. Open the [`content/_data/sidebar.json`](/content/_data/sidebar.json) file and
-   add a link the new `privatestate` page. In this example, we're adding a
-   command, so we'll add a link underneath the `api` section.
-
-```diff
-"api": [
-  {
-    "title": "API",
-    "slug": "api",
-    "children": [
-      {
-        "title": "Commands",
-        "slug": "commands",
-        "children": [
-          {
-            "title": "and",
-            "slug": "and"
-          },
-          {
-            "title": "as",
-            "slug": "as"
-          },
-          // ...
-+         {
-+           "title": "privateState",
-+           "slug": "privatestate"
-+         }
-        ]
-      }
-    ]
-  }
-]
-```
-
-The `sidebar.json` file contains a tree-like structure of nodes. Each node
-contains a `title`, `slug`, and optionally a `redirect` property. The URLs for
-each node in the `sidebar.json` are determined by the placement of each node in
-the hierarchy. For example, the `privateState` node that we added in the
-previous example would have a generated URL of `/api/commands/privatestate`
-since `privateState` is a child of the `commands` node, which is a child of the
-`api` node. If the `privateState` node were to contain a `redirect` instead of a
-`slug` property, it would link to the path used as the `redirect`:
-
-```json
-{
-  "title": "privateState",
-  "redirect": "/guides/overview/why-cypress"
-  // creates a link to the `/guides/overview/why-cypress` page
-}
-```
-
-> Note: If a node contains a `redirect` property, there is no need to add a
-> `slug` property. The `redirect` property will always take precedence over the
-> `slug`.
-
-The ability to nest nodes inside of other nodes gives you the ability to create
-a menu structure up to three (3) levels deep:
-
-```json
-"api": [
-  {
-    "title": "API",
-    "slug": "api",
-    "children": [
-      {
-        "title": "Commands",
-        "slug": "commands",
-        "children": [
-          {
-            "title": "Custom Commands",
-            "slug": "custom-commands",
-            "children": [
-              {
-                "title": "Writing a Custom Command",
-                "slug": "writing-a-custom-command"
-                // this node should not contain any children
-              }
-            ]
-          }
-        ]
-      }
-    ]
-  }
-]
-```
-
-### Deleting Pages
-
-To delete a page:
-
-- Delete the page from the relevant directory under [`content`](/content).
-- Remove the link from the the [`sidebar.json`](/content/_data/sidebar.json).
-- Build the documentation site locally so that you can visually inspect and make
-  sure it was properly deleted.
+To add a page, such as a new guide or API documentation check out how to do so
+[here](https://docusaurus.io/docs/create-doc)
 
 ### Writing the Changelog
 
+<!-- TODO: this needs to be updated once we determine how to handle the new changlog -->
```

Comment:
This PR needs to be updated once we figure out how we are handling the changelog.